Tools/autotest/param_metadata/param_parse.py

```diff
@@ -439,8 +439,25 @@ def process_library(vehicle, library, pathprefix=None):
 
     debug("Processed %u documented parameters" % len(library.params))
 
+
+def natural_sort_key(libname):
+    """Natural sort key used for sorting alphanumeric strings"""
+
+    # splitting string into parts (numeric , non-numeric)
+    parts = re.split(r'(\d+)', libname)
+
+    key = []
+    for part in parts:
+        if part.isdigit():
+            key.append(int(part))
+        else:
+            key.append(part.lower())
+
```

Comment:
```suggestion
    return tuple((int(p) if p.isdigit() else p) for p in parts)
```

Don't think we want the lower case.